pkg/controller/node/status.go

```diff
@@ -89,6 +89,10 @@ func calculateStatus(pool *mcfgv1.MachineConfigPool, nodes []*corev1.Node) mcfgv
 		mcfgv1.SetMachineConfigPoolCondition(&status, *supdated)
 		supdating := mcfgv1.NewMachineConfigPoolCondition(mcfgv1.MachineConfigPoolUpdating, corev1.ConditionTrue, "", fmt.Sprintf("All nodes are updating to %s", pool.Spec.Configuration.Name))
 		mcfgv1.SetMachineConfigPoolCondition(&status, *supdating)
+
+		if pool.Spec.Paused == true { // if the pool is Paused and updating, the pool will never progress.
+			glog.Warningf("Pool %s is Paused, but is 'Updating'.  Pool will not progress", pool.Name)
```

Comment:
Is there a reason you put it here? Why not just put it where we already check for paused?: https://github.com/openshift/machine-config-operator/blob/cd925586f71d709a36a11945a7abf8104e3d2046/pkg/controller/node/node_controller.go#L723

Reply:
Also I don't think this should be glog.Warning(), could just be glog.Info

Reply:
Can we get this comment into 1 sentence? "Pool %s is paused and will not update/progress/<insert word of choice"."

Reply:
@kikisdeliveryservice thanks for that feedback.  I went ahead and committed those changes.  I did not consider putting this change around where we checked for 'Paused'.  That would make sense too and can certainly make that change if you want.

Reply:
if you can move & delete comment that would be great.

Reply:
sure thing.  done!